dev-lang/lua/lua-5.3.3.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="A powerful, fast, light-weight, embeddable scripting language"
+DESCRIPTION="
+Lua combines simple procedural syntax with powerful data description \
+constructs based on associative arrays and extensible semantics. Lua is \
+dynamically typed, runs by interpreting bytecode for a register-based virtual \
+machine, and has automatic memory management with incremental garbage \
+collection, making it ideal for configuration, scripting, and rapid prototyping.
+"
+HOMEPAGE="http://www.lua.org"
+SOURCE_URI="http://www.lua.org/ftp/lua-$portVersion.tar.gz"
+CHECKSUM_SHA256="5113c06884f7de453ce57702abaac1d618307f33f6789fa870e87a59d772aca2"
+LICENSE="MIT"
+COPYRIGHT="1994-2015, Lua.org, PUC-Rio"
+REVISION="1"
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```

Comment:
Can you please add a blank line before ARCHITECTURES and swap x86 with x86_gcc2 ?

Maybe you can also add x86_gcc2 to SECONDARY_ARCHITECTURES.